sample/src_polymer/elements/elements.html

```diff
@@ -30,3 +30,6 @@
 <link rel="import" href="scene-perception-pages/scene-perception-page.html">
 <link rel="import" href="switch-image-button/switch-image-button.html">
 <link rel="import" href="toggle-image-button/toggle-image-button.html">
+
+<script src='../libs/stats.min.js'></script>
+<script src='../common/utils.js'></script>
```

Comment:
Eh, Add  `<script>` directly in elements.html will contaminate the js into every page? Even for some pages having no need of it.